Packs/CarbonBlackEnterpriseEDR/Integrations/CarbonBlackEnterpriseEDR/README.md

```diff
@@ -1,85 +1,53 @@
-## Overview
----
-
-VMware Carbon Black Enterprise EDR is an advanced threat hunting and incident response solution delivering continuous visibility for top security operations centers (SOCs) and incident response (IR) teams. (formerly known as ThreatHunter)
+VMware Carbon Black Enterprise EDR (formerly known as Carbon Black ThreatHunter) is an advanced threat hunting and incident response solution delivering continuous visibility for top security operations centers (SOCs) and incident response (IR) teams. (formerly known as ThreatHunter)
+This integration was integrated and tested with version 1.1.30 of Carbon Black Enterprise EDR
 
 ## Configure VMware Carbon Black Enterprise EDR on Cortex XSOAR
----
 
-1. Navigate to __Settings__ \> __Integrations__ \> __Servers & Services__.
+1. Navigate to **Settings** > **Integrations** > **Servers & Services**.
 2. Search for VMware Carbon Black Enterprise EDR.
-3. Click __Add instance__ to create and configure a new integration instance.
-    * __Name__: a textual name for the integration instance.
-    * __Server URL (e.g. https://defense.conferdeploy.net)__
-    * __Organization Key__
-    * __Custom Key__
-    * __Custom ID__
-    * __Fetch incidents__
-    * __Incident type__
-    * __Trust any certificate (not secure)__
-    * __Use system proxy settings__
-    * __First fetch timestamp (\<number\> \<time unit\>, e.g., 12 hours, 7 days, 3 months, 1 year)__
-    * __Fetch limit__
-4. Click __Test__ to validate the URLs, token, and connection.
+3. Click **Add instance** to create and configure a new integration instance.
+
+    | **Parameter** | **Required** |
+    | --- | --- |
+    | Server URL (e.g. https://defense.conferdeploy.net) | True |
+    | Organization Key | True |
+    | Custom Key | False |
+    | Custom ID | False |
+    | Custom ID | False |
+    | Custom Key | False |
```

Comment:
Are there instructions somewhere for getting this information?
Why is Custom ID listed twice?